test/ngAnimate/animateSpec.js

```diff
@@ -1543,6 +1543,49 @@ describe("animations", function() {
       });
     });
 
+    they('should not animate an element when the pinned element (which is the $prop element), is pinned to an element that is not a child of the $rootElement',
```

Comment:
Wasn't it at 100 chars ? Did that change and I missed that ?
Imo, 100 chars makes sense because it makes side-by-side diffing much easier.